src/core/controllers/viewport-controls.js

```diff
@@ -35,7 +35,7 @@ const EVENT_TYPES = {
   KEYBOARD: ['keydown', 'keyup']
 };
 
-export default class Controls {
+export default class ViewportControls {
```

Comment:
@Pessimistress has requested that avoid the use of the word controls as it is being used e.g. in react-map-gl to talk about other things.

Reply:
It is fine, we can discuss a good name and change it later on, here we just make the class name, file name and import name are all the same.